addons/dex/readme.md

```diff
@@ -9,6 +9,6 @@ Please refer to [Dex's Website](https://dexidp.io/docs/) for more details.
 
 ## Versions
 
-* 0.6.6
```

Comment:
Please leave the 0.6.6 changelog and append 0.7.0 changelog

Reply:
done